universe/space/space_attributes.go

```diff
@@ -324,6 +332,10 @@ func (s *Space) RemoveSpaceAttribute(attributeID entry.AttributeID, updateDB boo
 		}
 	}()
 
+	if s.world != nil {
+		s.world.GetCalendar().OnAttributeRemove(attributeID)
+	}
```

Comment:
same 
https://github.com/momentum-xyz/ubercontroller/blob/develop/universe/space/space_attributes.go#L395

Reply:
hooked registered in new place